openapi-v1.yaml

```diff
@@ -4816,9 +4816,9 @@ definitions:
     description: Contains the latest schema and all schemas for the opened array
     type: object
     properties:
-      latest_array_schema:
```

Comment:
Shall we add a comment here, explaining that this must match what core expects?

Reply:
Right, I was thinking along the same lines, how to avoid that very nasty issue in the future (great job finding it!!).

Since there is no automated way, maybe adding a comment in this file in a very visible way and updating our own documentation could help a bit: https://github.com/TileDB-Inc/TileDB-Cloud-REST/blob/7c23501a9d7d491bc09028b5cb8c223d68c306c7/docs/UpgradeCapnpSchemas.md